README.markdown

```diff
@@ -179,60 +182,32 @@ Get a specific member's information (open/click rates etc.) from MailChimp:
 gibbon.lists(list_id).members(lower_case_md5_hashed_email_address).retrieve
 ```
 
-### Batch Operations
+#### Fields
```

Comment:
Let's leave out the e-commerce examples. The point of this readme is not to explain how to use the MailChimp's API. That's what MailChimp's API docs are for. The batch operations are a bit trickier to set up, which is why they're an exception. The other examples are there for illustrative purposes and because they are common cases.